test/src/server/routes/auth.js

```diff
@@ -0,0 +1,65 @@
+import {createEditor, truncateEntities} from '../../../test-helpers/create-entities';
+import app from '../../../../src/server/app';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import {get} from 'lodash';
+import orm from '../../../bookbrainz-data';
+
+
+chai.use(chaiHttp);
+const {expect} = chai;
+const {Editor} = orm;
+
+const deletedUserAttrib = {
+	areaId: null,
+	bio: '',
+	cachedMetabrainzName: null,
+	genderId: null,
+	metabrainzUserId: null,
+	revisionsApplied: 0,
+	revisionsReverted: 0,
+	titleUnlockId: null,
+	totalRevisions: 0
+};
+
+function verifyDeletedUser(user) {
+	if (!get(user, 'name', '').includes('Deleted User')) { return false; }
+	for (const key in deletedUserAttrib) {
+		if (Object.hasOwnProperty.call(deletedUserAttrib, key)) {
+			if (get(user, key) !== get(user, key, null)) {
+				return false;
+			}
+		}
+	}
+	return true;
+}
+describe('delete user', () => {
+	const adminEditorId = 1;
+	const adminEditorName = 'UserDeleter';
+	const userEditorId = 2;
+	const userEditorName = 'NormalUser1';
+	const otherEditorId = 3;
+	const otherEditorName = 'NormalUser2';
+	let agent;
+	before(async () => {
+		await createEditor(adminEditorId, {name: adminEditorName});
+		await createEditor(userEditorId, {name: userEditorName});
+		await createEditor(otherEditorId, {name: otherEditorName});
+		agent = chai.request.agent(app);
+		await agent.get('/cb');
+	});
+	after(truncateEntities);
+	it('Normal User should not be able to delete other users', async () => {
+		const res = await agent.post(`/delete-user/${otherEditorName}?access_token=${userEditorId}`);
+		expect(res.ok).to.be.false;
+		// unauthorized
+		expect(res.status).to.be.equal(401);
+	});
```

Comment:
I'd also love to see, for good measure, another test to ensure we get a 401 for an unauthenticated user.
Basically the same test as the above, but using `await chai.request(app).post(…`  <- this will not use a user session because we won't call `/cb` as we do in the `before` function